src/command_modules/azure-cli-hdinsight/azure/cli/command_modules/hdinsight/_help.py

```diff
@@ -9,7 +9,7 @@
 
 helps['hdinsight'] = """
     type: group
-    short-summary: Manage HDInsight clusters.
+    short-summary: Manage HDInsight clusters, applications, extensions, OMS, script actions and usages.
```

Comment:
You can probably just have `Manage Azure HDInsight resources`. Customers will be able to see the subgroups with `az hdinsight -h`.

Reply:
Sure. Will fix.